ozw-network-visualization-card.js

```diff
@@ -129,13 +129,20 @@ class OZWNetworkVisualizationCard extends HTMLElement {
     }
   }
 
+  //Set different shapes for different device types
   _getShape(device) {
     if (device.node_basic_string === "Static Controller") {
+      return "triangle";
+    } else if (device.node_generic_string === "Binary Switch") {
       return "box";
-    } else if (device.node_basic_string === "Routing Slave") {
+    } else if (device.node_generic_string === "Multilevel Switch") {
       return "ellipse";
-    } else {
+    } else if (device.node_generic_string === "Multilevel Sensor") {
       return "circle";
+    } else if (device.node_generic_string === "Binary Sensor") {
+      return "circle";
+    } else {
+      return "database";
```

Comment:
I don't think we should change shape based on the device sub-type. The graph is to view connectivity, and that is influenced mostly by if the device is a router or not, so shape should represent that. I don't see the in having different shapes for switches, covers, lights, etc in a connectivity graph, while quickly seeing if the device is a router or not helps.

Reply:
I can leave it on my own version, I don't mind, but it makes it much easier for me to locate nodes. If you did see my screenshot of my network, there are many nodes, and some differentiator helps me locate devices. I would have preferred colours for different types, and will make that change locally eventually, but time is limited, so the shapes was a quickfix.

BTW: thanks for the commit-suggestion tip.

Reply:
So I see there is a "Changes Requested" below, which I assume is related to this /\, so do I accept by "Resolve conversation" and does that commit your original code and drop mine?

Reply:
Regarding the shapes, the thing is that there are a lot of different zwave device types, and I don't think we should have shapes for all of them :D Also, we can just put the device type as text and then you can use the search box to highlight all of the same type. In the future maybe we can have different colors for different types indeed, but I'll prefer to leave shapes for routing/not routing.

---
The "Changes requested" is just my "review" status asking for changes to the PR. For accepting the code suggestions, check this: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

Reply:
So, if you are ok with restoring the shape for now, I think we can merge this :) 
I'll try to do the different colors for types this week.

Reply:
Yes, okay for me.  Hopefully ready to incorporate. 

Ps: the search box is very helpful now I know what it is for.

Reply:
Btw, just improved the visibility of highlighted nodes and added some hint to the search box to make it more noticeable.